src/scripts/patch_shell.lua

```diff
@@ -12,4 +12,14 @@ print("patch_shell start")
   'ScriptCB_SetIFScreen' is used infrequently though.
   (Prefer to do this in the '0' addme though)
 ]]
+
+if IsFileExist == nil then 
+	print("patch_shell: defining IsFileExist")
+	IsFileExist = function(path)
+		local testPath = "..\\..\\".. path
```

Comment:
What is the purpose of explicitly stating the path changes like this instead of allowing the developer to specify it? As a developer, the expected behaviour would be for it to function the same as any other file-related BF2 function. Or is this for some CC-specific reason?

Reply:
The intent here is to have  'IsFileExist()' pair with 'dofile()' .

Because 'dofile()' works relative to the "Battle" or "Gamedata" folder and 'ScriptCB_DoFile()' works relative to "data\\_LVL_PC" or "data2\\_lvl_common" folder.

So one can do:
```LUA
if IsFileExist(luaFilePath) == 1 then
     dofile(luaFileName)
end
```
instead of:
```LUA
if ScriptCB_IsFileExist("..\\..\\" .. luaFilePath) ==1 then
     dofile(luaFileName)
end
```